bosh/opsfiles/production.yml

```diff
@@ -0,0 +1,75 @@
+- type: replace
+  path: /name?
+  value: shibboleth-production
+
+- type: replace
+  path: /instance_groups/name=idp/properties/idp?
+  value: 
+    login:
+      forgotPasswordLink: https://account.fr.cloud.gov/forgot-password
+    totp:
+      issuerName: "cloud.gov"
+    entityID: https://idp.fr.cloud.gov
+    scope: fr.cloud.gov
+    metadata:
+      providers:
+      - id: fr
+        metadataURL: https://login.fr.cloud.gov/saml/metadata
+    sslPrivateKey: ((idp_cert.private_key))
```

Comment:
Looks like these are copied and pasted between ops files. Can they go in the base manifest?

Reply:
Sure, though it breaks the "override the idp property" thing above, which was why I did it this way.

Reply:
I think copying and pasting isn't ideal, and smaller operations are generally good, so yeah, I would recommend rearranging.